client/webserver/site/src/js/forms.ts

```diff
@@ -1058,7 +1058,7 @@ export class WalletWaitForm {
     Doc.show(wallet.synced ? page.syncCheck : wallet.syncProgress >= 1 ? page.syncSpinner : page.syncUncheck)
     Doc.show(wallet.balance.available > fee.amount ? page.balCheck : page.balUncheck)
 
-    page.progress.textContent = String(Math.round(wallet.syncProgress * 100))
+    page.progress.textContent = (wallet.syncProgress * 100).toFixed(1)
```

Comment:
I honestly don't think the third sig fig does anything for us or the users. How about changing `Math.round` to `Math.floor`. In combination with your `if (prog >= 0.999)` block, we should get the desired behavior.

Reply:
On mainnet and test, it sometimes slows to a crawl for whatever reason, like one crap peer slowing things down, and 45.2 changing to 45.7 gives you a sign of life.
Maybe most users just want a whole number?  I feel like one decimal is fine personally, but whatever you guys decided is fine with me.

Reply:
I personally like 3rd figure because of the sense of progress **chap** mentioned (I think it is noticeable on slow network), plus it is already implemented and working (according to my testing), so I wouldn't wanna change it again unless you insist ;)